getting started/crowdsale.sol

```diff
@@ -0,0 +1,76 @@
+contract token { mapping (address => uint) public balance; function token() {}  function sendToken(address receiver, uint amount) returns(bool sufficient) {  } }
```

Comment:
You could use `function token();` here to make it explicit that it is only an interface.